sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -761,6 +762,39 @@ export class JsonRpcProvider extends Provider {
     }
   }
 
+  //Dynamic Fields
+  async getDynamicFields(parent_object_id: ObjectId, cursor: ObjectId | null = null, limit: number | null = null): Promise<DynamicFieldPage> {
+    try {
+      const resp = await this.client.requestWithType(
+        'sui_getDynamicFields',
+        [parent_object_id, cursor, limit],
+        DynamicFieldPage,
+        this.options.skipDataValidation
+      );
+      return resp;
+    } catch (err) {
+      throw new Error(
+        `Error getting dynamic fields with request type: ${err}`
```

Comment:
nit: can we also log parameters like other existing methods: e.g., `parent_object_id`, `cursor`, `limit`, etc..